### PR TITLE
Update LICENSE to MPLv2

### PR DESCRIPTION
**Pull request by prestosole**

### CODE REVIEW

**Comment by triallax** [comment]
For us to use MPLv2 properly, we have to add headers to each source code file. Please add a header to one source code file so I can review it if necessary and then you can add it to the rest of the files.

**Comment by prestosole** [comment]
```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 ```

https://www.mozilla.org/en-US/MPL/headers/

**Comment by triallax** [comment]
@PrestoSole sounds good to me.

File: LICENSE [hunk 1]
```diff
@@ -1,29 +1,375 @@
...
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
-FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
+Clima
+
```
Comment (triallax) [inline LICENSE]: We don't need this line (well, not for now, but it can be added when it's needed). Remove it for now.

**Review by triallax — CHANGES_REQUESTED** [review]

**Comment by triallax** [comment]
You also need to change the license tile in the about page.

**Comment by prestosole** [comment]
Resolved!

**Comment by prestosole** [comment]
Don't merge this PR now though. I don't want to cause conflicts for #220

**Pull request closed without merging**